feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/mappers/TransactionMappers.kt

```diff
@@ -1,102 +1,130 @@
 package jp.co.soramitsu.feature_wallet_impl.data.mappers
 
-import jp.co.soramitsu.core_db.model.TransactionLocal
+import jp.co.soramitsu.core_db.model.OperationLocal
 import jp.co.soramitsu.feature_wallet_api.data.mappers.mapTokenTypeLocalToTokenType
 import jp.co.soramitsu.feature_wallet_api.data.mappers.mapTokenTypeToTokenTypeLocal
+import jp.co.soramitsu.feature_wallet_api.domain.model.Operation
 import jp.co.soramitsu.feature_wallet_api.domain.model.Token
-import jp.co.soramitsu.feature_wallet_api.domain.model.Transaction
 import jp.co.soramitsu.feature_wallet_api.domain.model.WalletAccount
 import jp.co.soramitsu.feature_wallet_api.domain.model.amountFromPlanks
-import jp.co.soramitsu.feature_wallet_api.domain.model.planksFromAmount
-import jp.co.soramitsu.feature_wallet_impl.data.network.model.response.TransactionRemote
-import jp.co.soramitsu.feature_wallet_impl.presentation.model.TransactionModel
+import jp.co.soramitsu.feature_wallet_impl.data.network.model.response.SubqueryHistoryElementResponse
 
-fun mapTransactionStatusToTransactionStatusLocal(status: Transaction.Status) = when (status) {
-    Transaction.Status.PENDING -> TransactionLocal.Status.PENDING
-    Transaction.Status.COMPLETED -> TransactionLocal.Status.COMPLETED
-    Transaction.Status.FAILED -> TransactionLocal.Status.FAILED
-}
 
-fun mapTransactionStatusLocalToTransactionStatus(status: TransactionLocal.Status) = when (status) {
-    TransactionLocal.Status.PENDING -> Transaction.Status.PENDING
-    TransactionLocal.Status.COMPLETED -> Transaction.Status.COMPLETED
-    TransactionLocal.Status.FAILED -> Transaction.Status.FAILED
+fun mapOperationStatusToOperationLocalStatus(status: Operation.Status) = when (status) {
+    Operation.Status.PENDING -> OperationLocal.Status.PENDING
+    Operation.Status.COMPLETED -> OperationLocal.Status.COMPLETED
+    Operation.Status.FAILED -> OperationLocal.Status.FAILED
 }
 
-fun mapTransactionToTransactionModel(transaction: Transaction): TransactionModel {
-    return with(transaction) {
-        TransactionModel(
+fun mapOperationToOperationLocalDb(operation: Operation, source: OperationLocal.Source): OperationLocal {
+    with(operation) {
+        val amount = this.operation.getOperationAmount()
+        val fee = this.operation.getOperationFee()
+
+        return OperationLocal(
             hash = hash,
-            type = tokenType,
-            senderAddress = senderAddress,
-            recipientAddress = recipientAddress,
-            accountName = accountName,
-            isIncome = isIncome,
-            date = date,
-            amount = amount,
-            status = status,
-            fee = fee,
-            total = total
+            address = address,
+            time = time * 1000,
```

Comment:
It is generally not a good idea to use "Magic constant"
I suggest to either use Duration API or to write our own extension (if there are none)

Reply:
Done

Reply:
Is it? It is still present in the latest changes

Reply:
😅 Now yes, it is